app/controllers/api/members_controller.rb

```diff
@@ -16,4 +22,13 @@ def create
   def member_params
     params.permit(:name, :email, :country, :postal)
   end
+
+  def member_validation
+    [
+      { name: 'email', data_type: 'email', required: true },
+      { name: 'country', data_type: 'country', required: false },
+      { name: 'postal', data_type: 'postal', required: false },
+      { name: 'name', data_type: 'text', required: true }
+    ]
+  end
```

Comment:
are you saying "12 LOC that are strictly related to the member creation process" is too many? but we're in the `create` method of the `MembersController`! that's a super reasonable place for member creation logic to live.